src/utils/constants.js

```diff
@@ -22,5 +22,13 @@ export const TEMPLATE_WORD_GROUPS = {
   animals3_insects_pd: "Animals 3"
 };
 
+// Firebase constants
+export const DB = {
```

Comment:
fam u really gonna call this const DB just make it more descriptive so u dont need that comment explaining what it is